ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -919,6 +919,278 @@ keys:
                 enabled:
                   type: bool
                   default: true
+  snmp_server:
+    type: dict
+    keys:
+      engine_ids:
+        type: dict
+        keys:
+          local:
+            type: str
+            convert_types:
+            - int
+            description: 'Engine id in hexadecimal
+
+              '
+          remotes:
+            type: list
+            items:
+              type: dict
+              keys:
+                id:
+                  type: str
+                  description: 'Engine ID In Hexadecimal
```

Comment:
On the local engine ID you have convert_types: int, shouldn't this be here as well?